### ISSUE TITLE
http://jerakia.io/ is down

### ISSUE THREAD

**Issue by mrjk** [jerakia/jerakia#155]
The main domain is not reachable anymore.

http://jerakia.io/

Is this software still maintained? If not, any good alternatives?

Cheers

**Comment by beddari** [comment]
No, it is not maintained. A lot of the ideas and knowledge contained here have been carried on in various ways, possibly most notably by @vincentbernat in a recent blog post https://vincent.bernat.ch/en/blog/2021-network-jerikan-ansible

Unless someone else picks it up and continues it I think perhaps we should archive it, to signal that fact? WDYT @crayfishx ?

What are you using currently for large diverse data sets @jtopjian ?

**Comment by vincentbernat** [comment]
The domain is on pending delete. I suppose it would cost more to salvage it now and unless we do it, spammers will take it over. Who is the current owner? Is it @crayfishx? Do we have a corporate user we could ask to pay for the domain?

**Comment by beddari** [comment]
I don't think the domain has a lot of value at this point, some might disagree though?

I think we should just focus on this github org, it has always been the first google hit for "jerakia" and could be a longer lasting point of reference for anyone interested.

By updating some of the READMEs with more current information and alternatives we could probably save some time for people?

**Comment by beddari** [comment]
Also we still got some free t-shirts 👍🏼 just ask me on Twitter hehe https://twitter.com/hashtag/jerakia
